templates/document.html

```diff
@@ -21,96 +21,102 @@
+                    <a href='mailto:info@grails.org'><img class='' src='[%url]/images/email.svg' alt='Email Icon'/></a>
                 </li>
                 <li>
-                    <a href='https://slack.grails.org'><img class='' src='[%url]/images/slack.svg' alt='Slack Icon' /></a>
+                    <a href='https://slack.grails.org'><img class='' src='[%url]/images/slack.svg'
+                                                            alt='Slack Icon'/></a>
                 </li>
                 <li>
-                    <a href='https://www.youtube.com/@GrailsFramework'><img class='' src='[%url]/images/youtube.svg' alt='Youtube Icon' /></a>
+                    <a hre f='https://www.youtube.com/@GrailsFramework'><img class='' src='[%url]/images/youtube.svg'
+                                                                             alt='Youtube Icon'/></a>
                 </li>
                 <li>
-                    <a href='https://www.linkedin.com/showcase/official-grails/'><img class='' src='[%url]/images/linkedin.svg' alt='LinkedIn Icon' /></a>
+                    <a href='https://www.linkedin.com/showcase/official-grails/'><img class=''
+                                                                                      src='[%url]/images/linkedin.svg'
+                                                                                      alt='LinkedIn Icon'/></a>
                 </li>
                 <li>
-                    <a href='https://github.com/grails/'><img class='' src='[%url]/images/github.svg' alt='Github Icon' /></a>
+                    <a href='https://github.com/grails/'><img class='' src='[%url]/images/github.svg'
+                                                              alt='Github Icon'/></a>
                 </li>
                 <li style="margin-bottom:10px;">
-                    <a href='https://twitter.com/grailsframework'><img class='' src='[%url]/images/x-twitter.svg' alt='Twitter Icon' /></a>
+                    <a href='https://twitter.com/grailsframework'><img class='' src='[%url]/images/x-twitter.svg'
+                                                                       alt='Twitter Icon'/></a>
                 </li>
             </ul>
         </nav>
-        <nav class='partnersnav'><ul>
-
-            <li>The Grails framework is Open Source
-                <a href="https://www.apache.org/licenses/LICENSE-2.0.html">Apache 2 License</a>
-            </li>
-
-            <li>
-                <a href='[%url]/privacy-policy.html'>Privacy Policy</a>
-            </li>
-            <li>
-                <a href='[%url]/trademark-policy.html'>Trademark Policy & Brand Guidelines</a>
-            </li>
-        </ul></nav>
     </div>
-</footer><div>
-    <script type='text/javascript'>
-        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
```

Comment:
Apache does not allow for Google tracking.

Reply:
We can use Mamoto analytics once we graduate.